make/photon/ui/Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM library/photon:1.0
-
-RUN mkdir /harbor/
+FROM vmware/photon:1.0
 
+RUN tdnf distro-sync -y \
```

Comment:
tdnf distro-sync || echo ?

@reasonerjt

Reply:
it's fine.